client/src/app/layout.tsx

```diff
@@ -17,6 +17,12 @@ export default async function RootLayout({
 }) {
   return (
     <html lang='en'>
+      <head>
+        <link
+          href='https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap'
```

Comment:
Esto creo que es lo que menos seguro estoy del PR, mi idea es importar la fuente Poppins al proyecto y esta es una forma que encontré viable (eslint me tira warning pero no error)
Otra forma más prolija que encontré era importar la font en globals.css pero no me funcionó.

Reply:
font-poppins no funca? tailwind debe tener ya esa fuente supongo y vi que lo agregaste al tailwind.config

Reply:
No se la banca no, por las dudas me basé en este ejemplo que muestra que hay que agregarlo tanto al tailwind.config como importar la fuente web desde la url https://www.tailwindtoolbox.com/guides/adding-fonts-to-tailwind-css

Reply:
Nop, no se la banca. Te paso los recursos que encontré como guía para agregar las fuentes.
https://www.tailwindtoolbox.com/guides/adding-fonts-to-tailwind-css
https://blog.logrocket.com/how-to-use-custom-fonts-tailwind-css/
En ambas paginas veo que más alla de la declaración dentro de tailwind de la fuente, además hay que ocuparse de importarla como fuente Web. (el segundo link usa particularmente Poppins)

Reply:
ahi va, pense que tailwind venia con esa fuente pero por lo que veo no, no me parece mal esa forma de importar la fuente